packages/mermaid/src/diagrams/info/infoRenderer.ts

```diff
@@ -11,40 +11,20 @@ import type { DrawDefinition, HTML, SVG } from '../../diagram-api/types.js';
  * @param version - MermaidJS version.
  */
 const draw: DrawDefinition = (text, id, version) => {
-  try {
-    log.debug('rendering info diagram\n' + text);
+  log.debug('rendering info diagram\n' + text);
```

Comment:
Do we need this line?

Reply:
Not really sure, my thought of leaving this line it's would give us the ability to track where the error might occur, i.e., if the diagram didn't get rendered and this line didn't get logged then it should somewhere before that.

Reply:
Totally fine, lets leave it there and deal with it later. No harm, it is already working anyway.